### PR TITLE
✨ Load userStats from like.co API

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

**Review by williamchong — APPROVED** [review]
Weird that you would have to refetch user stat going from portfolio to share page, but lets start with this PR for now

**Review by nwingt — APPROVED** [review]

**Review by nwingt — COMMENTED** [review]
Seems fetching user stats in `<UserStatsBase/>` instead of portfolio/dashboard/share page is more convenient?

**Comment by WeiJun0827** [comment]
> Seems fetching user stats in `<UserStatsBase/>` instead of portfolio/dashboard/share page is more convenient?

Was thinking of the pattern that letting the component(`<UserStatsBase>` here) fetch data it uses or allowing the page fetch data and then inject it into the component.

File: src/mixins/portfolio.js [hunk 1]
```diff
@@ -214,6 +216,10 @@ export default {
         this.currentTab = 'collected';
       }
     },
+    async updateUserStats(address) {
+      const { data } = await this.$api.get(getUserNFTStats(address));
+      this.userStats = data;
+    },
```
Comment (nwingt) [inline src/mixins/portfolio.js:222]: Why not put inside `<UserStatsBase>`?

Comment (nwingt) [reply src/mixins/portfolio.js:222]: And add a watcher for `statWallet` to refresh the data

Comment (nwingt) [reply src/mixins/portfolio.js:222]: For this pattern, the parent does not need to know the details of fetch & data structure, however, the parent can set `ref="userStats"` in `<UserStatsBase />`, then fetch the data using `this.$refs.userStats.fetch()`

Comment (nwingt) [reply src/mixins/portfolio.js:222]: Maybe `<UserStatsBase/>` can rename to `<UserStatsController />`

Comment (WeiJun0827) [reply src/mixins/portfolio.js:222]: Indeed make sense.

File: src/components/UserStats/Controller.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      return this.userStats?.createdCollectorCount || 0;
+    },
+    isLoadingStats() {
+      return !this.userStats;
+    },
+  },
+  watch: {
+    statWallet() {
+      if (statWallet) {
+        this.getUserStats();
```
Comment (nwingt) [inline src/components/UserStats/Controller.vue:53]: ??
```suggestion
        this.updateUserStats();
```

File: src/components/UserStats/Controller.vue [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+export default {
+  mixins: [portfolio],
+  props: {
+    tag: {
+      type: String,
+      default: 'div',
+    },
+    statWallet: {
+      type: String,
+      default: '',
```
Comment (nwingt) [inline src/components/UserStats/Controller.vue:25]:
```suggestion
      required: true,
```

**Review by nwingt — CHANGES_REQUESTED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**